api/app/morecast_v2/forecasts.py

```diff
@@ -105,3 +109,91 @@ def filter_for_api_forecasts(forecasts: List[WeatherIndeterminate], actuals: Lis
         if actual_exists(forecast, actuals):
             filtered_forecasts.append(forecast)
     return filtered_forecasts
+
+
+def get_fwi_values(actuals: List[WeatherIndeterminate], forecasts: List[WeatherIndeterminate]) -> Tuple[List[WeatherIndeterminate], List[WeatherIndeterminate]]:
+    """
+    Calculates actuals and forecasts with Fire Weather Index System values by calculating based off previous actuals and subsequent forecasts.
+
+    :param actuals: List of actual weather values
+    :type actuals: List[WeatherIndeterminate]
+    :param forecasts: List of existing forecasted values
+    :type forecasts: List[WeatherIndeterminate]
+    :return: Actuals and forecasts with calculated fire weather index values
+    :rtype: Tuple[List[WeatherIndeterminate], List[WeatherIndeterminate]
+    """
+    all_indeterminates = actuals + forecasts
+    indeterminates_dict = defaultdict(dict)
+
+    # Shape indeterminates into nested dicts for quick and easy look ups by station code and date
+    for indeterminate in all_indeterminates:
+        indeterminates_dict[indeterminate.station_code][indeterminate.utc_timestamp.date()] = indeterminate
+
+    for idx, indeterminate in enumerate(all_indeterminates):
+        last_indeterminate = indeterminates_dict[indeterminate.station_code].get(
+            indeterminate.utc_timestamp.date() - timedelta(days=1), None)
+        if last_indeterminate:
```

Comment:
```suggestion
        if last_indeterminate is not None:
```

Just a suggestion for the sake of clarity